src/component/__tests__/Timer.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Timer from '../Timer';
+import renderer from 'react-test-renderer';
+
+describe('Timer', () => {
+  jest.useFakeTimers();
+  const finishTime = jest.fn();
+
+  it('renders the Timer', () => {
+    const tree = renderer.create(<Timer/>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('calls tick every 1000 miliseconds', () => {
+    const TimerComponent = renderer.create(<Timer time = {0} finishTime = {finishTime}/>);
+
+    expect(setTimeout).toHaveBeenCalledWith(expect.any(Function), 1000);
+    expect(setTimeout).toHaveBeenCalledTimes(1);
+
+    TimerComponent.update();
+
+    expect(setTimeout).toHaveBeenCalledWith(expect.any(Function), 1000);
+    expect(setTimeout).toHaveBeenCalledTimes(2);
```

Comment:
Is there a way we could test the desired side effect? (The timer rendering a number that changes)

I don't think this test provides us a ton of value as is; we could call `setTimeout` with anything under the sun and this would still pass.

Full disclosure I'm not sure how we would accomplish this.